website/docs/plugin/framework/functions/parameters/index.mdx

```diff
@@ -94,3 +94,7 @@ Parameter names are used in runtime errors to highlight which parameter is causi
 │
 │ Invalid value for "bool_param" parameter: a bool is required.
 ```
+
+## Parameter Validation
```

Comment:
Should the parameter-based validation documentation be here instead since value and type validations are already covered in `website/docs/plugin/framework/handling-data/types/custom.mdx`? 

It's a little awkward since we don't have any specific page for function-specific validations to serve as an index but there is not enough specific info for functions that isn't already covered in `website/docs/plugin/framework/validation.mdx` or `website/docs/plugin/framework/handling-data/types/custom.mdx`

Reply:
Fair point. I've added some documentation to cover implementation of `validation.ValidateableParameter` interface and linked through to custom types.